### PR TITLE
Run a two-steps tobiko workflow

### PR DESCRIPTION
**Pull request by eduolivares**
The test-operator periodic job podified-multinode-edpm-deployment-crc will run a tobiko workflow with two test steps.

### CODE REVIEW

**Comment by lpiwowar** [comment]
@eduolivares, FYI, I've tested the workflows here -> https://github.com/openstack-k8s-operators/test-operator/pull/58

It seems to be working but additional change is needed on the ci-framework side. Right now we collect stdout only from one pod [1][2].

[1] https://logserver.rdoproject.org/58/58/c5175fd3777cff2e6393ccaa8241b740ee51454b/github-check/podified-multinode-edpm-deployment-crc/3793439/controller/ci-framework-data/tests/test_operator/
[2] https://github.com/openstack-k8s-operators/ci-framework/blob/319bce54c09867cd73155c0f95ef0d81c4d5b78f/roles/test_operator/tasks/run-test-operator-job.yml#L127C1-L127C49

**Comment by lpiwowar** [comment]
Also, the question is whether we want to collect the stdout using the test_operator role at all because the stdout is already being collected by must-gather -> https://logserver.rdoproject.org/58/58/c5175fd3777cff2e6393ccaa8241b740ee51454b/github-check/podified-multinode-edpm-deployment-crc/3793439/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/tobiko-tests-step1-workflow-step-0-mkvw5/logs/tobiko-tests.log

Previously, the tempest / tobiko role needed to collect the logs because the execution was not part of the openshift deployment.

File: .zuul.yaml [hunk 1]
```diff
@@ -28,7 +28,12 @@
                   service_available.swift false
                   service_available.cinder false
               cifmw_run_tobiko: true
-              cifmw_test_operator_tobiko_testenv: 'functional -- tobiko/tests/functional/podified/test_topology.py'
+              cifmw_test_operator_tobiko_workflow:
+                - stepName: 'podified-functional'
+                  testenv: 'functional -- tobiko/tests/functional/podified/test_topology.py'
```
Comment (lpiwowar) [inline .zuul.yaml:33]:
```
cp: target '/var/lib/tobiko/external_files/tobiko-tobiko-tests-workflow-step-0/' is not a directory
```
The issue is probably the `--` in the name of the testenv [1].  

[1] https://logserver.rdoproject.org/61/61/f6d4ad5e7af4032384e64f28b33a22a0fae53b02/github-check/podified-multinode-edpm-deployment-crc/a78b181/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/tobiko-tests-podified-functional-workflow-step-0-sn6v6/logs/tobiko-tests.log

Comment (lpiwowar) [reply .zuul.yaml:33]: Posible fix? -> https://github.com/openstack-k8s-operators/tcib/pull/153

**Comment by eduolivares** [comment]
> Also, the question is whether we want to collect the stdout using the test_operator role at all because the stdout is already being collected by must-gather -> https://logserver.rdoproject.org/58/58/c5175fd3777cff2e6393ccaa8241b740ee51454b/github-check/podified-multinode-edpm-deployment-crc/3793439/controller/ci-framework-data/logs/openstack-k8s-operators-openstack-must-gather/namespaces/openstack/pods/tobiko-tests-step1-workflow-step-0-mkvw5/logs/tobiko-tests.log
> 
> Previously, the tempest / tobiko role needed to collect the stdout because the execution was not part of the openshift deployment.

Ok, thank you. I didn't know you were already working on this.
It looks good to me to remove the collection of the container output file from the test_operator role, considering it's included in the must-gather artifacts.

**Comment by lpiwowar** [comment]
recheck

**Comment by lpiwowar** [comment]
@eduolivares, can you please remove the WIP flag if the running job passes and change the patch so that it is mergable, when you have time? I think it would be good we merge this and have the workflow covered in the periodic job.

**Comment by eduolivares** [comment]
> @eduolivares, can you please remove the WIP flag if the running job passes and change the patch so that it is mergable, when you have time? I think it would be good we merge this and have the workflow covered in the periodic job.

OK!
If the job passes, I will fix the patch (enable tempest again, remove the check pipeline), so that it can be approved.

**Comment by eduolivares** [comment]
@lpiwowar 
The results were good: https://review.rdoproject.org/zuul/build/8e14e2e46fe14eed92cbbdcd274fdc32
This PR is ready to review.

**Pull request merged**